src/test/java/integrationTests/mojo/environmentSetup/AgentSetup.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package integrationTests.mojo.environmentSetup;
+
+import org.apache.maven.plugin.logging.SystemStreamLog;
+import org.junit.rules.TemporaryFolder;
+import org.mule.tools.client.agent.AgentClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.junit.Assert.fail;
+
+public class AgentSetup {
+
+  private Logger log;
+  private static final long TEN_SECONDS = 10000;
+  private static final String MULE_HOME_FOLDER_PREFIX = "/mule-enterprise-standalone-";
+  private static final String MULE_VERSION = "3.8.1";
```

Comment:
shouldn't this be a parameter?
At some point in time we may want to test against different mule versions